docs/collections/graphs/create-graph-edge.md

```diff
@@ -0,0 +1,25 @@
+---
+title: Create Graph Edge
+sidebar_position: 20
+---
+
+The Graph Edge collection is a form of document collection. It is used define relationships between two documents by using the **_from** and **_to** system attributes and the document **_id** system attribute.
```

Comment:
@diana-macrometa So far we've been using "edge" as an improper noun, do we want to reconsider this and maybe use "Edge" as a proper noun?

Reply:
No, I'm using "edge" as they use it throughout the section. It's generic in this usage.